### PR TITLE
TNT-40895 Fix incorrect MaxAge set in response cookies

### PR DESCRIPTION
**Pull request by XDex**
## Description
Fix incorrect MaxAge set in response cookies

https://jira.corp.adobe.com/browse/TNT-40895

### CODE REVIEW

File: src/main/java/com/adobe/target/edge/client/utils/TargetConstants.java [hunk 1]
```diff
@@ -15,5 +15,5 @@
 
   public static final String COOKIE_NAME = "mbox";
   public static final String CLUSTER_COOKIE_NAME = "mboxEdgeCluster";
```
Comment (artur-ciocanu) [inline src/main/java/com/adobe/target/edge/client/utils/TargetConstants.java:17]: Could we move this to a separate PR. We need an automated way to generate and include these details here, either via class loader resources or something similar.

**Review by artur-ciocanu — APPROVED** [review]
Great work! Let's fix the version number in a separate PR.

File: src/main/java/com/adobe/target/edge/client/utils/TargetConstants.java (hunk 1 above)
Comment (XDex) [reply src/main/java/com/adobe/target/edge/client/utils/TargetConstants.java:17]: As discussed, we should remove this version from code and just read it directly from gradle.properties, which should be copied to src/main/resources during build so it gets packaged in the JAR.
I'll create a separate task for this
